discretize/__init__.py

```diff
@@ -25,9 +25,10 @@
 
             to compile the cython code.
             """
-            )
+        )
+from discretize import tests
 
-__version__   = '0.5.1'
-__author__    = 'SimPEG Team'
-__license__   = 'MIT'
-__copyright__ = '2013 - 2019, SimPEG Developers, http://simpeg.xyz'
+__version__ = "0.5.1"
+__author__ = "SimPEG Team"
+__license__ = "MIT"
+__copyright__ = "2013 - 2020, SimPEG Developers, http://simpeg.xyz"
```

Comment:
Should we just remove 2020 here? For the licensing, really only the initial year is needed (that way we don't have to keep updating it...)

Reply:
Sure!